contents/tutorials/nuxt-analytics.md

```diff
@@ -0,0 +1,282 @@
+  <div>
+    <h1>About</h1>
+    <NuxtLink to="/">Back Home</NuxtLink>
+  </div>
+</template>
+```
+
+Lastly, replace the code in `app.vue` with:
+
+```vue file=app.vue
+<template>
+  <div>
+    <NuxtLayout>
+      <NuxtPage/>
+    </NuxtLayout>
+  </div>
+</template>
+```
+
+The basic setup is now complete. Run `npm run dev` to see your app.
+
+![Basic Nuxt app](../images/tutorials/nuxt-analytics/basic-app.png)
+
+## Adding PostHog on the client side
+
+> This tutorial shows how to integrate PostHog with `Nuxt 3`. If you're using `Nuxt 2`, see [our Nuxt docs](/docs/libraries/nuxt-js) for how to integrate.
+
+With our app set up, it’s time to install and set up PostHog. If you don't have a PostHog instance, you can [sign up for free](https://us.posthog.com/signup). 
+
+First install `posthog-js`:
+
+```bash
+npm install posthog-js
+```
+
+Then, add your PostHog API key and host to your `nuxt.config.ts` file. You can find your project API key in your [PostHog project settings](https://app.posthog.com/settings/project)
+
+```ts file=nuxt.config.ts
+export default defineNuxtConfig({
+ devtools: { enabled: true },
+  runtimeConfig: {
+    public: {
+      posthogPublicKey: '<ph_project_api_key>',
+      posthogHost: '<ph_instance_address>'
+    }
+  }
+})
+```
+
+Create a new [plugin](https://nuxt.com/docs/guide/directory-structure/plugins) by creating a new folder called `plugins` in your base directory and then a new file `posthog.client.js`:
+
+```bash
+mkdir plugins
+cd plugins 
+touch posthog.client.js
+```
+
+Add the following code to your `posthog.client.js` file:
+
+```js file=plugins/posthog.client.js
+import { defineNuxtPlugin } from '#app'
+import posthog from 'posthog-js'
+
+export default defineNuxtPlugin(nuxtApp => {
+  const runtimeConfig = useRuntimeConfig();
+  const posthogClient = posthog.init(runtimeConfig.public.posthogPublicKey, {
+    api_host: runtimeConfig.public.posthogHost,
+  })
+  
+  return {
+    provide: {
+      posthog: () => posthogClient
+    }
+  }
+})
+```
+
+Once you’ve done this, reload your app. You should begin seeing events in the [PostHog events explorer](https://us.posthog.com/events).
+
+<ProductScreenshot
+  imageLight={EventsLight} 
+  imageDark={EventsDark} 
+  alt="Events in PostHog" 
+  classes="rounded"
+/>
+
+### Capturing pageviews
+
+You might notice that moving between pages only captures a single pageview event. This is because PostHog only captures pageview events when a [page load](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) is fired. Since Nuxt creates a single-page app, this only happens once, and the Nuxt router handles subsequent page changes.
+
+If we want to capture every route change, we must write code to capture pageviews that integrates with the router.
+
+In `posthog.client.js`, set up PostHog to capture pageviews in the `router.afterEach` function. Additionally, you can use `nextTick` so that the capture event fires only after the page is mounted.
```

Comment:
```suggestion
In `posthog.client.js`, set up PostHog to capture pageviews in the `router.afterEach` function. Additionally, you can use `nextTick` so that you capture the event after the page is mounted.
```